datahub-web-react/src/app/policy/PoliciesPage.tsx

```diff
@@ -197,35 +253,193 @@ export const PoliciesPage = () => {
+                        <Button
+                            disabled={!record?.editable}
+                            onClick={() => onToggleActiveDuplicate(record?.policy)}
+                            style={{ color: record?.editable ? 'green' : ANTD_GRAY[6] }}
+                        >
+                            ACTIVATE
+                        </Button>
+                    )}
+                    <Button
+                        disabled={!record?.editable}
+                        onClick={() => onRemovePolicy(record?.policy)}
+                        type="text"
+                        shape="circle"
+                        danger
+                    >
+                        <DeleteOutlined />
+                    </Button>
+                </ActionButtonContainer>
+            ),
+        },
+    ];
+
+    const tableData = policies?.map((policy) => ({
+        urn: policy?.urn,
+        policy,
+        name: policy?.name,
+        type: policy?.type,
+        description: policy?.description,
+        allUsers: policy?.actors?.allUsers,
+        allGroups: policy?.actors?.allGroups,
+        resolvedGroups: policy?.actors?.resolvedGroups,
+        resolvedUsers: policy?.actors?.resolvedUsers,
+        state: policy?.state,
+        editable: policy?.editable,
+        privileges: policy?.privileges,
+        resources: policy?.resources,
+    }));
+
     return (
         <SearchablePage>
             {policiesLoading && <Message type="loading" content="Loading policies..." />}
             {policiesError && message.error('Failed to load policies :(')}
             {updateError && message.error('Failed to update the Policy :(')}
             <PoliciesContainer>
-                <PoliciesTitle level={2}>Manage Policies</PoliciesTitle>
-                <NewPolicyButton onClick={onClickNewPolicy} data-testid="add-policy-button">
-                    + New Policy
-                </NewPolicyButton>
-                <PolicyList
-                    bordered
-                    dataSource={policies}
-                    renderItem={(item: unknown) => (
-                        <PolicyListItem policy={item as Policy} onView={() => onViewPolicy(item as Policy)} />
-                    )}
-                />
-                <PaginationContainer>
-                    <Pagination
-                        style={{ margin: 40 }}
-                        current={page}
-                        pageSize={pageSize}
-                        total={totalPolicies}
-                        showLessItems
-                        onChange={onChangePage}
-                        showSizeChanger={false}
-                    />
-                </PaginationContainer>
+                <PoliciesHeaderContainer>
+                    <PoliciesTitle level={2}>Manage Policies</PoliciesTitle>
+                    <Typography.Paragraph type="secondary">
+                        Manage access for users & groups of DataHub using Policies.
```

Comment:
"Manage access for Users & Groups of DataHub using Policies."

Reply:
Changed and committed